pkg/flowcli/gateway/grpc.go

```diff
@@ -31,6 +31,10 @@ import (
 	"github.com/onflow/flow-cli/pkg/flowcli/project"
 )
 
+// MaxGRPCMessageSize 16mb, matching the value set in onflow/flow-go
+// https://github.com/onflow/flow-go/blob/master/utils/grpc/grpc.go#L5
+const MaxGRPCMessageSize = 1024 * 1024 * 16
```

Comment:
Can we make this private? Just trying to keep the external package interface small.